atlas/src/main/java/us/ihmc/atlas/multisenseBlobExperiments/MultiSenseBlobDetectionTestModule.java

```diff
@@ -66,17 +65,17 @@ public void run()
             while (latestBufferedImage == null)
                ThreadTools.sleep(10);
 
-            ImagePanel imagePanel = ShowImages.showWindow(latestBufferedImage, "Circle Detector");
-            JFrame frame = (JFrame) SwingUtilities.getWindowAncestor(imagePanel);
+//            ImagePanel imagePanel = ShowImages.showWindow(latestBufferedImage, "Circle Detector");
+//            JFrame frame = (JFrame) SwingUtilities.getWindowAncestor(imagePanel);
```

Comment:
What's going on here? I would prefer enclosing these in a boolean of "visualize" or whatever, rather than commenting them out. Or is this to remove the JFrame dependency?

Reply:
ImagePanel and ShowImages come from boofcv.